SpecFlowProjectMars/Pages/LoginPage.cs

```diff
@@ -15,15 +15,14 @@ public class LoginPage
         IWebElement logInButton;
         public void LoginActions(IWebDriver driver, string username, string password)
```

Comment:
please do not pass the driver through action methods instead extend to Common driver